### DISCUSSION TITLE
群组添加文章

### DISCUSSION THREAD

**Discussion by LHT-Liu** [zotero-chinese/wiki#117] [求助 / Q&A] [answered]
<img width="1071" alt="image" src="https://github.com/zotero-chinese/wiki/assets/64388319/0cccaf2f-2a1a-49e2-afa7-1fde4a472541">
群组中添加文章，不能把pdf直接拖动吗？

**Comment by northword** [comment]
![image](https://github.com/zotero-chinese/wiki/assets/44738481/5b0b0b6f-1e6a-4769-a2c4-8d82d259d82c)

**Reply by LHT-Liu** [reply]
说实话，我没太懂，不好意思

**Reply by northword** [reply, marked as answer]
你可能没有你这个群组的文件权限——可能是下面 zeping 说的你这是一个“公开群组”（公开群组不能存文件），也可能是（私有群组）你没有打开文件存储权限。

**Comment by zepinglee** [comment]
大致是，“Public Open”的群组不能存储文件。

<img width="353" alt="Screenshot 2023-12-24 at 11 02 01" src="https://github.com/zotero-chinese/wiki/assets/12290822/bbeef3af-9a15-4836-8003-25eef2a7020e">

应该是因为公开阅读的话会触及出版商的利益。

<img width="429" alt="Screenshot 2023-12-24 at 11 02 22" src="https://github.com/zotero-chinese/wiki/assets/12290822/556cdf24-08e3-418b-86a9-916a0a0fabc0">

**Reply by LHT-Liu** [reply]
感谢，我明白了
